md/howto-debugcustompage.md

```diff
@@ -0,0 +1,89 @@
+# How to Debug a custom page ? 
+
+<h2></h2>
+
+##  Question
+
+The engine load the custom page, and then you want to debug it : change an item in the Look And Feel, modify the Groovy code or whatever. The engine has a mode that you can do that directly, and just click on the portal to reload the page and take into account the change without zip and deploy your change.
```

Comment:
Bonita BPM Engine comes with a mode that allows direct edition and live reload of the page in the Portal to see the changes.